wercker.yml

```diff
@@ -1,4 +1,4 @@
-box: node:4.0.0
+box: node:6.0.0
```

Comment:
Why's this needed? Happy to bump if code really needs it, but if not can we move it to a separate PR to keep this one super focused.